README.md

```diff
@@ -0,0 +1,52 @@
+# Static Site Generator App Extension for Quasar, the Vue.js Framework
+
+## Warning
+This extension for Quasar is an early-stage extension and under active development. Use at your own risk.
+
+## Development/Testing
+If you're interested in assisting with testing or development, you can get started using the following steps:
+
+1. cd into your quasar directory or run
+    ````bash
+    quasar create test-ext && cd test-ext
+    ````
+1. Install a couple dependencies:
+    ````bash
+    yarn add --dev https://github.com/freddy38510/quasar-app-extension-ssg critters-webpack-plugin
+    ````
+1. Manually add the extension:
+    ````bash
+    quasar ext add ssg
+    ````
+    You might want to say no to using critter; there may be an issue with that option.
+1. Configure your quasar app to use the `vueRouterMode: 'history'` rather than `hash` in `quasar.conf.js`.
+1. If you don't have a second page defined in `src/router/routes.js`, do so now, e.g.,
```

Comment:
What is the goal of this step ? The extension is not working with only one page ?

Reply:
In fact we should recommend to create a route for catching 404 since our website will operate as a SPA for subsequent navigations.
```javascript
// src/router/routes.js
{ path: '*', component: () => import('pages/error404.vue') }
```

For the very first navigation to the website or refresh, if no "index.html" is found, we have a fallback SPA '404.html' by default.
By "fallback SPA", i mean the '404.html' file is not a static generated page, instead it's a generated page in SPA mode.
This should be handled by the server running the app. Netlify and others uses "404.html" as custom 404 page.